src/pages/index.js

```diff
@@ -189,6 +189,7 @@ const api = new Api({
   },
 }); // log the error to the console
```

Comment:
```js


  const profileImage = document.querySelector(".profile__image");
 api.updateProfileImage(inputValues).then((res) => {
  profileImage.src = inputValues.link;
```


in `index.js` in line 159

Class `UserInfo` should set the entire user’s data (including the **avatar**)